addon/templates/components/bs4/bs-form/element/errors.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if show}}
+  <div class={{feedbackClass}}>{{messages.firstObject}}</div>
+{{/if}}
```

Comment:
It's more a preference than a rule. Either works. I didn't feel it was worth differentiating.